test/functional/apps/dashboard/dashboard_filter_bar.js

```diff
@@ -124,5 +124,23 @@ export default function ({ getService, getPageObjects }) {
         await pieChart.expectPieSliceCount(1);
       });
     });
+
+    describe('saved search filtering', function () {
+      before(async () => {
+        await filterBar.ensureFieldEditorModalIsClosed();
+        await PageObjects.dashboard.gotoDashboardLandingPage();
+        await PageObjects.dashboard.clickNewDashboard();
+        await PageObjects.dashboard.setTimepickerInDataRange();
+      });
+
+      it('are added when pie chart legend item is clicked', async function () {
+        await dashboardAddPanel.addVisualization('Rendering Test: pie');
+        await PageObjects.dashboard.waitForRenderComplete();
+        await pieChart.filterByLegendItem('4,886');
```

Comment:
Just noticed this doesn't actually catch the bug (I just hit it again in a PR that passed) because it's using a pie slice, not a saved search. :)